src/MCMCDiagnosticTools.jl

```diff
@@ -16,18 +17,19 @@ using Statistics: Statistics
 
 export bfmi
 export discretediag
-export ess_rhat, ess_rhat_bulk, ess_tail, rhat_tail, ESSMethod, FFTESSMethod, BDAESSMethod
+export ess, ess_rhat, ESSMethod, FFTESSMethod, BDAESSMethod
 export gelmandiag, gelmandiag_multivariate
 export gewekediag
 export heideldiag
 export mcse
 export rafterydiag
+export rhat
```

Comment:
Maybe this export could be moved to `ess` and `ess_rhat` above, since they seem to belong together?